src/main/resources/requests/utils/axiosConfig.ts

```diff
@@ -1,4 +1,5 @@
 import axios from 'axios';
 
 export const dataManagementApi = axios.create({baseURL: (process.env.DATA_MANAGEMENT_URL || 'https://pip-data-management.staging.platform.hmcts.net/'), timeout: 10000});
+export const subscriptionManagementApi = axios.create({baseURL: (process.env.SUBSCRIPTION_MANAGEMENT_URL || 'https://pip-subscription-management.staging.platform.hmcts.net/'), timeout: 10000});
```

Comment:
not sure what you mean. Can you be more specific please?

Reply:
You have created a environment variable on line 4 called `SUBSCRIPTION_MANAGEMENT` however you have either not created that env variable or forgot to add the file that initialises it so it will never have a value

Reply:
I have implemented SUBSCRIPTION_MANAGEMENT_URL like DATA_MANAGEMENT_URL. Maybe i have missed something but I can't see DATA_MANAGEMENT_URL added somewhere else. I thought that for this variable to work we need to add it to a different repo.

Reply:
so your comment should not be part of this PR then